Kudu.Core/Deployment/DeploymentManager.cs

```diff
@@ -544,8 +544,8 @@ private DeployResult GetResult(string id, string activeDeploymentId, bool isDepl
                 string repositoryRoot = _environment.RepositoryPath;
                 var perDeploymentSettings = DeploymentSettingsManager.BuildPerDeploymentSettingsManager(repositoryRoot, _settings);
 
-                string delayMaxInStr = perDeploymentSettings.GetValue(SettingsKeys.MaxRandomDelayInSec, false);
-                if (delayMaxInStr != null)
+                string delayMaxInStr = perDeploymentSettings.GetValue(SettingsKeys.MaxRandomDelayInSec, true);
```

Comment:
why did it become true :) just curious. btw, by named parameter I meant
```csharp
perDeploymentSettings.GetValue(SettingsKeys.MaxRandomDelayInSec, onlyPerSite: true);
```
VS will tell you that it's not needed and you should consider removing it, but we like them in kudu. They make reading code easier :)